projects/plugins/beta/class-jetpack-beta-admin.php

```diff
@@ -168,12 +168,30 @@ public static function to_test_content() {
 			case 'pr':
 				return self::to_test_pr_content( $branch );
 			case 'master':
+				return self::to_test_general_rules_content();
```

Comment:
I thought that `master` referred to `bleeding edge`. If we wanted the instructions to come up for the `latest stable` as well, should we set `self::to_test_general_rules_content();` as a default case?